www/js/main.js

```diff
@@ -96,8 +96,29 @@ angular.module('emission.main', ['emission.main.recent', 'emission.main.diary',
     }
 })
 
-.controller('ControlCtrl', function($scope, $window, $ionicScrollDelegate, $state, $ionicPopup, $ionicActionSheet, $ionicPopover, $rootScope) {
+.controller('ControlCtrl', function($scope, $window, $ionicScrollDelegate, $state, $ionicPopup, $ionicActionSheet, $ionicPopover, $rootScope, ControlHelper) {
+    $scope.emailLog = ControlHelper.emailLog;
     $scope.dark_theme = $rootScope.dark_theme;
+
+    
+    $scope.toggleLowAccuracy = function() {
+        if ($scope.settings.collect.config == null) {
+            return false; // config not loaded when loading ui, set default as false
```

Comment:
Do u want me to debug on this? I tested that they are not loaded on first call. However, the reason that I dare to set a default value is that I noticed the toggle function is called lots of times, and in the later calls, the config is loaded and the toggle is set to the right state. I'm only using this default to avoid null pointer exception.